requests/src/requests/Requester.scala

```diff
@@ -54,7 +54,7 @@ object Requester{
 }
 case class Requester(verb: String,
                      sess: BaseSession){
-
+  val CMD = verb.toUpperCase // allow submitting as lower case ...
```

Comment:
Why are you adding `CMD` as a field here?

Reply:
"verb" was basically used as an enum, only without insisting it be case dependent. In general it was accessed using by comparing "POST" or something to verb.toUpperCase. One place in the code it was accessed without uppercasing, which I took to be a bug. Maybe I'm old-school, but I don't like code that does the same thing over and over even if the compiler will probably optimize it away, so I changed it to do the upper-casing once and compare to CMD. Maybe there is a style you prefer, but I would fix that bug in any case.